site/layouts/shortcodes/youtube-gallery.html

```diff
@@ -1,19 +1,18 @@
 <div class="youtube-gallery-wrapper">
   {{- $apiKey := getenv "HUGO_GOOGLE_YT_API_KEY" -}}
+  {{- $playlistId := (.Get "id") -}}
   {{ if $apiKey }}
-    {{- $videos := resources.GetRemote (printf "https://www.googleapis.com/youtube/v3/playlistItems?key=%s&part=snippet,contentDetails&maxResults=5&playlistId=PLMZDRUOi3a8NtMq3PUS5iJc2pee38rurc" $apiKey) | transform.Unmarshal -}}
+    {{- $videos := resources.GetRemote (printf "https://www.googleapis.com/youtube/v3/playlistItems?key=%s&part=snippet,contentDetails&maxResults=5&playlistId=%s" $apiKey $playlistId) | transform.Unmarshal -}}
     <div class="youtube-gallery-cards">
-      {{- range where $videos.items "snippet.title" "!=" "Deleted video" | first 3 -}}
-        {{- if ne .snippet.title "Deleted video" -}}
+      {{- range where $videos.items "snippet.title" "!=" "Private video" | first 3 -}}
```

Comment:
Ideally, we should have a better filter logic here and check for `Private video` and `Deleted video`. But I couldn't convince Hugo to do that. With the recent playlist changes, I changed things to `Private Video` because it was showing up in the list.